frontend/game/game.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 
-requirejs(['./board/board.js', './hand.js', './helper.js'], function (board, hand, helper) {
+requirejs(['./board/board.js', './hand.js', './scores.js', './helper.js'], function (board, hand, scores, helper) {
 
     function setState(state) {
         hand.setHand(state["hand"]);
         board.setBoard(state["rows"]);
+	scores.setScores(state["scores"]);
```

Comment:
indentation is one space off 🙃

Reply:
vim is annoying here...
it autoindents javascript using \t

Reply:
```
set autoindent
set expandtab
set shiftwidth=4
set softtabstop=4
```

although i'm considering starting to push for the "tabs for indentation, spaces for alignment" philosophy in future projects.